Patch-Source: https://cgit.git.savannah.gnu.org/cgit/pspp.git/patch/?id=d43ce274a2a623a2207d4be3fbe0d5b544c2532f
---
From d43ce274a2a623a2207d4be3fbe0d5b544c2532f Mon Sep 17 00:00:00 2001
From: Friedrich Beckmann <friedrich.beckmann@tha.de>
Date: Tue, 3 Mar 2026 08:02:45 +0100
Subject: configure: remove check if gettext version is greater 0.20

This check failed for gettext version 1.0. I removed the check
as I think this is obsolete today.
---
 acinclude.m4 | 24 ------------------------
 configure.ac |  6 ------
 2 files changed, 30 deletions(-)

diff --git a/acinclude.m4 b/acinclude.m4
index 7e262c561..f85f80742 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -397,27 +397,3 @@ AC_DEFUN([PSPP_CHECK_DOT],
        pspp_cv_dot=no
      fi])
    AM_CONDITIONAL([HAVE_DOT], [test "$pspp_cv_dot" = yes])])
-
-
-dnl Check that xgettext is sufficiently recent
-dnl Before calling this macro, AC_CHECK_PROGS([XGETTEXT], [xgettext]) should have been called,
-dnl which sets the ac_cv_prog_XGETTEXT cache variable.
-AC_DEFUN([PSPP_CHECK_XGETTEXT],
- [AC_CACHE_CHECK([whether gettext is sufficiently recent],
-   [pspp_cv_progrecent_xgettext],
-   [pspp_cv_progrecent_xgettext=maybe
-    ver=`$ac_cv_prog_XGETTEXT --version | sed -n -e '/xgettext/s/[[^0-9]]*\([[0-9\.]][[0-9\.]]*\).*/\1/p'`
-    case $ver in # (
-      *.*.*) ;; # (
-      *.*) ver="$ver.0" ;;
-    esac
-    majmin=`echo $ver | sed -e 's/\.[[0-9]]*$//'`
-    major=`echo $majmin | sed -e 's/\.[[0-9]]*$//'`
-    minor=`echo $majmin | sed -e 's/^[[0-9]]*\.//'`
-    if test "$major" -lt 0; then
-      pspp_cv_progrecent_xgettext=no;
-    elif test "$minor" -lt 20; then
-      pspp_cv_progrecent_xgettext=no
-    else
-      pspp_cv_progrecent_xgettext=yes
-    fi])])
diff --git a/configure.ac b/configure.ac
index 74224b7d2..f7843c9b2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -87,12 +87,6 @@ if test "x$XGETTEXT" = x; then
   PSPP_REQUIRED_PREREQ([xgettext (is usually provided with gettext)])
 fi
 
-PSPP_CHECK_XGETTEXT
-
-if test "$pspp_cv_progrecent_xgettext"  != "yes"; then
-   PSPP_REQUIRED_PREREQ([xgettext version 0.20 or greater])
-fi
-
 dnl Checks for libraries.
 AC_SYS_LARGEFILE
 AC_SEARCH_LIBS([sin], [m])
-- 
cgit v1.2.3

